src/components/Audiences/injectProcessDestinations.js

```diff
@@ -10,48 +10,59 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { cookieJar, noop } from "../../utils";
+import { noop } from "../../utils";
 
 const createResultLogMessage = (urlDestination, success) => {
   return `URL destination ${success ? "succeeded" : "failed"}: ${
     urlDestination.spec.url
   }`;
 };
 
-const processUrls = (fireReferrerHideableImage, logger, destinations) => {
-  const urlDestinations = destinations.filter(dest => dest.type === "url");
-
-  return Promise.all(
-    urlDestinations.map(urlDestination => {
-      return fireReferrerHideableImage(urlDestination.spec)
-        .then(() => {
-          logger.info(createResultLogMessage(urlDestination, true));
-        })
-        .catch(() => {
-          // We intentionally do not throw an error if destinations fail. We
-          // consider it a non-critical failure and therefore do not want it to
-          // reject the promise handed back to the customer.
-          logger.error(createResultLogMessage(urlDestination, false));
-        });
-    })
-  ).then(noop);
-};
-
-const processCookies = destinations => {
-  const cookieDestinations = destinations.filter(
-    dest => dest.type === "cookie"
-  );
+export default ({
+  fireReferrerHideableImage,
+  logger,
+  cookieJar,
+  isPageSsl
+}) => {
+  const extraCookieOptions = isPageSsl
+    ? { sameSite: "none", secure: true }
```

Comment:
Aren't these options returned by the Konductor if it's https?

Reply:
They are for "state:store" handles, but not "activation:push" handles.